docs: document issue with coraza default body parser (#123)
coreruleset/nextcloud-rule-exclusions-plugin: README.md

    ctl:ruleRemoveTargetById=922130;MULTIPART_PART_HEADERS"
```

### Coraza default body processor

> By default Coraza will use the `URLENCODED` body processor when no other request body parser has been configured, what body parser Coraza uses is defined by the `coraza.conf` recommended file. This behavior is not desirable for Nextcloud since Nextcloud is used to upload files which may include raw binary data or pretty much anything. This behavior can be changed by using the `RAW` body processor when no other parser has been configured.

Reviewer: ```suggestion
By default, Coraza will use the `URLENCODED` body processor when no other request body processor has been configured. Which body processor Coraza uses is defined in the `coraza.conf` file. The default behavior is not desirable for Nextcloud since Nextcloud is used to upload files which may include raw binary data or pretty much anything. The following rule can be used to enable the `RAW` body processor when no other parser has been configured.
```

```
SecRule REQBODY_PROCESSOR "!@rx (?:URLENCODED|MULTIPART|XML|JSON)" \

>     "id:9508033,\

Reviewer: Is this the ID you want users to use? I'm pretty sure that users will simply copy the rule and the ID will stick. So it should be an ID that you won't use anywhere else accidentally.

Author: Yes it should be fine, the first 100 rules of plugins are reserved for initialization and configuration. These rule ids are pretty much never touched within the plugin so I think it's fine to use them within the readme.
